solved_exercises/dani/quadratic.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from cmath import sqrt
+
+
+def polynome(a, b, c):
+    """This function calculates values of a univariate quadratic equation
+    x1 = -b + sqrt(b^2 - 4ac)
+    x2 = -b - sqrt(b^2 - 4ac)
+    """
+    if not all(isinstance(i, (int, complex, float)) for i in [a, b, c]):
```

Comment:
can you read this out loud, in english ?

Reply:
You are using a negated condition on a group of conditions built with list comprehension. Far too many things at once, please break this down.